File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/repository/GradesResultRepository.java
package org.ilyatyamin.yacontesthelper.repository;

import org.ilyatyamin.yacontesthelper.dao.GradesResult;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface GradesResultRepository extends JpaRepository<GradesResult, Long> {
}

File: web-service/yacontest-helper/src/main/java/org/ilyatyamin/yacontesthelper/grades/repository/GradesResultRepository.kt
package org.ilyatyamin.yacontesthelper.grades.repository

import org.ilyatyamin.yacontesthelper.grades.dao.GradesResult
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface GradesResultRepository : JpaRepository<GradesResult, Long>
